src/neo-vm/OpCode.cs

```diff
@@ -441,9 +437,21 @@ public enum OpCode : byte
         /// </summary>
         FROMALTSTACK = 0x6C,
         /// <summary>
+        /// Duplicates the item on top of alt stack and put it on top of main stack.
+        /// </summary>
+        DUPFROMALTSTACK = 0x6D,
```

Comment:
I don't think this should be done.. why changing opcode 0x6A, and in a PR named "Add PUSHNULL" ? This will break most tutorials I've written, auxiliar neovm codes.. and for what reason?

Reply:
this breaks everything, including historical connections with BitcoinVM https://neoresearch.io/nvm-learn/#stack-operations

Reply:
It doesn't break anythink because it is for 3.0.

Reply:
I know it is NeoVM 3, which is different from 2, but it matters for teaching. Everything that is preserved, helps us a little bit.